user_docs/en/changes.t2t

```diff
@@ -4,6 +4,8 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and
+Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This release also drops support for Adobe Flash.

```

Comment:
Use "mathematics" in international contexts as the abbreviation [differs by English variety](https://en.wiktionary.org/wiki/math#Noun_2). Also fix a/an.

```suggestion
Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is an Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This release also drops support for Adobe Flash.

```

Reply:
Thanks Bill, I've made that change, and another grammatical error someone pointed out via email.